core-bundle/src/Resources/views/Frontend/preview_toolbar_base_js.html.twig

```diff
@@ -180,9 +187,19 @@
 </style>
 <script>
     (function () {
+
+        const toolbarWrapper = document.createElement('cto-toolbar');
+        document.querySelector('body').prepend(toolbarWrapper);
+
+        const shadow = toolbarWrapper.attachShadow({mode: 'open'});
+        toolbarWrapper.appendChild(shadow);
+
+        const css = document.querySelector('style[cto-toolbar]');
+        shadow.innerHTML = css.outerHTML;
```

Comment:
```suggestion
        toolbarWrapper.shadowRoot.appendChild(Array.from(document.querySelectorAll('style')).at(-1));
```

Reply:
I guess that could go wrong if there is anything else injecting a `<style>` into the body?